CHANGELOG.rst

```diff
@@ -2,9 +2,17 @@
 Changelog
 =========
 
+4.0.0 (unreleased)
```

Comment:
A V4 already exists for CMS 4: https://github.com/divio/djangocms-text-ckeditor/blob/support/4.0.x/djangocms_text_ckeditor/__init__.py#L2

Reply:
Yeah that might now be a clash given that ckeditor should now be bumped from 3.9 to 4.0 instead of 3.10. Can your branch be bumped to 5.x? Ofc. I can also just go with 3.10 but I prefer to do a major bump.